scripts/mutation-triage.sh

```
@@ -0,0 +1,85 @@
+  local id="$1"
+  local file="$2"
+  local search="$3"
+  local replace="$4"
+  local test_cmd="$5"
+
+  echo "=== $id ==="
+  cp "$file" "${file}.bak"
+  sed -i "s|${search}|${replace}|" "$file"
+
+  local exit_code=0
+  eval "$test_cmd" >/tmp/mut-${id}.log 2>&1 || exit_code=$?
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)